runtime/include/jvmti.h.m4

```diff
@@ -744,7 +744,10 @@ ifelse(eval(JAVA_SPEC_VERSION >= 11), 1, [	JVMTI_EVENT_SAMPLED_OBJECT_ALLOC = 86
 ifelse(eval(JAVA_SPEC_VERSION >= 19), 1, [	JVMTI_EVENT_VIRTUAL_THREAD_START = 87,], [dnl])
 ifelse(eval(JAVA_SPEC_VERSION >= 19), 1, [	JVMTI_EVENT_VIRTUAL_THREAD_END = 88,], [dnl])
 
-	JVMTI_MAX_EVENT_TYPE_VAL = 86,
+ifelse(eval(JAVA_SPEC_VERSION >= 19), 1, [	JVMTI_MAX_EVENT_TYPE_VAL = 88,],[
+ifelse(eval(JAVA_SPEC_VERSION >= 11), 1, [	JVMTI_MAX_EVENT_TYPE_VAL = 86,],[
+	JVMTI_MAX_EVENT_TYPE_VAL = 84,
+])])
```

Comment:
do we need the trailing comma `JVMTI_MAX_EVENT_TYPE_VAL = 84,` ?

Also in other places `])])` is not on a separate line

Reply:
there is an example in line 60 on how this is done

Reply:
the comma separates this line with the next line of `jvmtiEventEnsureWideEnum = 0x1000000`
fixed the brackets